config/initializers/yaml_renderer.rb

```diff
@@ -1,4 +1,7 @@
 ActionController::Renderers.add :yaml do |obj, _|
-  data = JSON.load(obj.to_json).to_yaml
+  # Create a serializable resource instance
+  serializable_resource = ActiveModelSerializers::SerializableResource.new(obj)
+
+  data = serializable_resource.as_json.to_yaml
```

Comment:
`as_json` isn't working here. It generates ruby hash, where keys are symbols.
```Ruby
user.as_json
 => {:id=>1, :handle=>"sonalkr132", :email=>"aditya.prakash132@gmail.com"}
user.as_json.to_yaml
 => "---\n:id: 1\n:handle: sonalkr132\n:email: aditya.prakash132@gmail.com\n" 
```
We want following behavior:
```Ruby
user.to_json
 => "{\"id\":1,\"handle\":\"sonalkr132\",\"email\":\"aditya.prakash132@gmail.com\"}"
JSON.parse(user.to_json).to_yaml
 => "---\nid: 1\nhandle: sonalkr132\nemail: aditya.prakash132@gmail.com\n"
```

We can't use `deep_stringify_keys` keys here because at times `obj` can be `Array` as well (for ex: `@rubygem.owners`). @bf4 any ideas?